flask_app.py

```diff
@@ -0,0 +1,148 @@
+"""
+I made a simple flask app that creates a SQLAlchemy database that stores simple information like the email, password and token.
+
+To make it work locally you have to install quite some modules.
+http://127.0.0.1:5000/
+
+The database is now filed with two user testcases, what you can do is delete the site.db from the data / database directory to test the user creation for yourself
```

Comment:
Avoid overly long lines even though Ruff won't complain about strings.
```suggestion
The database is now filed with two user testcases, what you can do is delete the site.db
from the data / database directory to test the user creation for yourself
```